assets/blocks/editor-components/toolbar-dropdown/index.js

```diff
@@ -24,62 +29,88 @@ import {
  * Dropdown for the editor toolbar.
  *
  * @param {Object}           props
- * @param {DropdownOption[]} props.options        Dropdown options.
- * @param {string}           [props.optionsLabel] Options label.
- * @param {Object}           props.icon           Icon for the toolbar.
- * @param {string}           props.value          Current dropdown value.
- * @param {Function}         props.onChange       Dropdown change callback, which receive
- *                                                the new value as argument.
+ * @param {DropdownOption[]} props.options             Dropdown options.
+ * @param {string}           [props.optionsLabel]      Options label.
+ * @param {Object}           props.icon                Icon for the toolbar.
+ * @param {string}           props.value               Current dropdown value.
+ * @param {Function}         props.onChange            Dropdown change callback, which receive the new value as argument.
+ * @param {Object}           props.toggleProps         Props passed to the toggle element.
+ * @param {Object}           props.popoverProps        Props passed to the popover component.
+ * @param {Function}         props.renderMenuItemProps Render function for a menu item. Should return a props object.
```

Comment:
I find the `renderMenuItemProps` a bit confusing as this dosn't only render. Maybe use `getMenuItemProps` or something similar?

Reply:
Yeah that sounds better 👍